FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt

```diff
@@ -30,7 +30,6 @@ libraries:
   + https://www.FreeRTOS.org/jobs/
   + https://www.FreeRTOS.org/ota/
   + https://www.FreeRTOS.org/pkcs11/
-  + https://www.FreeRTOS.org/mqtt_lts/
```

Comment:
Reason for removing from Beta1 readme.txt?

The same file also mentions about the Beta2 demos

Reply:
It was not there before as far as I know. I got added in the earlier PR.

Reply:
I did not mean to add it to this file in #152. However, if it's meant to be here too, then I can push an update to keep it

Reply:
Added it back. This way the READMEs will be identical, as they were before #152